core/src/epicli/data/common/ansible/playbooks/roles/elasticsearch_curator/tasks/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
-- name: Include installation tasks
-  include_tasks: install-es-curator.yml
+- name: Include installation task
+  include_tasks: install-es-curator-{{ ansible_os_family }}.yml
 
 - name: Include configuration tasks
   include_tasks: configure-cron-jobs.yml
```

Comment:
Small change but here we should add new line in line 6

Reply:
I added just to avoid "missing newline" warning